cfgov/v1/blocks.py

```diff
@@ -30,3 +33,39 @@ class Meta:
         handler = None
         method = 'POST'
         icon = 'form'
+
+class AnchorLink(blocks.StructBlock):
+    link_id = blocks.CharBlock(required=False, label='ID for this content block', help_text=(
```

Comment:
More nitpicky style: backend [style discussions](https://github.com/cfpb/back-end/issues/14) have roughly converged on trying to use 80-character line widths, and [PEP8](https://www.python.org/dev/peps/pep-0008/) for most other things like indentation.

It can be useful to run a tool like [flake8](https://pypi.python.org/pypi/flake8) (`pip install flake8`) against changed files (`flake8 cfgov/v1/blocks.py`, for example) to check code style.